docker-compose.yml

```diff
@@ -409,3 +409,9 @@ services:
       - "logging=true"
     restart: always
     logging: *default-logging
+  download-url-warning:
+    image: lblod/download-url-warning-service:0.0.2
```

Comment:
any reason we start from `0.0.2`? Usually we start a service off with `0.1.0`

Reply:
mmh I think just me not being aware of that convention haha, will fix that